controllers/tagsController.js

```diff
@@ -0,0 +1,34 @@
+import models from '../models';
+import ResponseHandler from '../helpers/responseHelper';
+import { STATUS } from '../helpers/constants';
+
+/**
+ * Class representing tags controller
+ *
+ * @class TagsController
+ */
+class TagsController {
+  /**
+   * Creates a new user resource
```

Comment:
#### Nitpick
- What user resource?

Reply:
That's a grammar typo, resolving changes right away.